common/main.c

```diff
@@ -5,19 +5,36 @@
 
 #include <compiler.h>
 #include <console.h>
+#include <debug.h>
 #include <dm.h>
 #include <stdbool.h>
 #include <work.h>
+#include <drivers/watchdog.h>
 #include <platform/devices.h>
 
+#define WDOG_TIMEOUT (5 * 1000 * 1000) /* 5 seconds */
```

Comment:
This needs to be in 24 MHz clock cycles, so 24x the current value.

Reply:
Fixed.